src/app.py

```diff
@@ -2218,7 +2219,7 @@ def get_associated_organs_from_dataset(id):
 """
 @app.route('/datasets/prov-info', methods=['GET'])
 def get_prov_info():
-    # String constants from app.cfg
+    # String constants
     GLOBUS_GROUPS_URL = 'https://raw.githubusercontent.com/hubmapconsortium/commons/test-release/hubmap_commons/hubmap-globus-groups.json'
```

Comment:
Let's remove this `GLOBUS_GROUPS_URL` since it's only used in the error message. Considering we have dev-integrate/master branches of commons being used on DEV/PROD, this URL may confuse us, also commons is published to PyPI now.